### PR TITLE
Give traitor chaplains clockwork cult stuff

### PR DESCRIPTION
**Pull request by MACMAN2003**
## About the PR

Gave traitor chaplains clockwork cult stuff in emagged pietyvends, the clockwork armor is identical to the blood cult armor. And added a clockwork spear (for mappers)
## Why / Balance

The nar'sie blood cult shouldn't be the only cult that traitor chaplains can pretend to be
## Technical details

n/a
## Media

- [x] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

image

## Breaking changes

n/a
**Changelog**

:cl:
- add: Traitor chaplains can now LARP as clockwork cultists.

### CODE REVIEW

File: Resources/Prototypes/Entities/Clothing/Head/hoods.yml [hunk 1]
```diff
@@ -97,6 +97,20 @@
...
+  name: clockwork hood
+  description: What is a clockwork cult without clockwork hoods?
+  components:
+  - type: Sprite
+    sprite: Clothing/Head/Hoods/clockwork_helmet_old.rsi
+  - type: Clothing
+    sprite: Clothing/Head/Hoods/clockwork_helmet_old.rsi
+  - type: Tag
+    tags:
+    - HidesHair
```
Comment (Agoichi) [inline Resources/Prototypes/Entities/Clothing/Head/hoods.yml:112]: Maybe you should add tag whitelistchameleon like cult have.

Comment (Agoichi) [reply Resources/Prototypes/Entities/Clothing/Head/hoods.yml:112]: Cause yo now... er... eh... well.. as i understand when you add single tag all parents tag are ignored

**Comment by EmoGarbage404** [comment]
i think these items are kinda strange since we don't have clockcult. Bloodcult items, while associated with the mode, are still fine and generic items without it. Having these feels kinda pointless without the actual mode behind it.

**Comment by UbaserB** [comment]
> i think these items are kinda strange since we don't have clockcult. Bloodcult items, while associated with the mode, are still fine and generic items without it. Having these feels kinda pointless without the actual mode behind it.

i might end up using it for a gimmick, like a ruin or wreck

File: Resources/Prototypes/Entities/Clothing/Head/hoods.yml (hunk 1 above)
Comment (TheShuEd) [reply Resources/Prototypes/Entities/Clothing/Head/hoods.yml:112]: Yeah, you need to add all parent tags

**Comment by mirrorcult** [comment]
this doesnt really make sense and just kind of conflicts if anyone wants to use clock cult stuff for other things (or adds clock cult itself)

**Comment by mirrorcult** [comment]
i already dont like that bloodcult stuff is used in random places and this kinda makes it worse

**Pull request closed without merging**